api/lib/application/users/user-controller.js

```diff
@@ -323,4 +324,12 @@ module.exports = {
     });
     return h.response(campaignParticipationForUserManagementSerializer.serialize(campaignParticipations));
   },
+
+  async findOrganizationsForActiveUserAdmin(request, h) {
```

Comment:
Remarque : 
- les admin du site pix admin sont des adminMember et pas des users
- en général, la notion de active ou les filtres utilisés se trouve dans le repo et la doc se trouve dans les assertions des associés, par exemple "it should not returns organization if user is deactivated"
- ça veut dire quoi active dans le contexte de ta PR ?

Reply:
Pour « actif », le besoin est le suivant : afficher un tableau avec la liste des orgas dans lequel ce user est un membre qui est ACTIF (= le champ memberships.disabledAt est NULL)

Reply:
Ici on va chercher les orga auxquelles appartient actuellement l'utilisateur (on ne va pas chercher les membres actifs d'une orga). Du coup j'aurai envie de partir du principe que `user's organization` c'est le terme pour désigner les orga dans lesquelles l'utilisateur est actif. Et dans le cas où j'ai besoin de parler des orga dans lesquelles l'utilisateur n'est pas actif : `disabled user's organization`, ou quelque chose du genre.

Reply:
Je me rends compte que j'avais mal compris. UserAdmin c'est pas les agents pix... Il est galère ce nommage. C'est quoi l'inverse de membership ? (si on part du principe que membership est le terme adéquat ^^)

Reply:
Le terme `Admin` dans les noms peut prêter à confusion. Est-ce qu'il ne vaut mieux pas faire une distinction entre les scopes `admin` et `mon-pix` ?